src/TestSuite/IntegrationTestTrait.php

```diff
@@ -1296,4 +1296,12 @@ protected function extractExceptionMessage(Exception $exception): string
             PHP_EOL .
             $exception->getTraceAsString();
     }
+
+    /**
+     * @return \Cake\TestSuite\TestSession
+     */
+    protected function getTestSession(): TestSession
```

Comment:
Should we call this `getSession()`? I don't know if the `Test` prefix helps much as the code is in a test case already.

Reply:
I wanted to be sure that people don't confuse this with the existing _requestSession.
But I am fine if we call the method a bit simpler.